JS/app.js

```diff
@@ -0,0 +1,123 @@
+// Book class: Represent book
+class Book {
+  constructor(title, author, pages, type) {
+    this.title = title;
+    this.author = author;
+    this.pages = pages;
+    this.type = type;
+  }
+}
+// UI class: Handle UI task
+class UI {
+  static displayBooks() {
+    const StoredBooks = [
+      {
+        title: 'Harry Potter',
+        author: 'John Doe',
+        pages: '234',
+        type: 'Read',
+      },
+      {
+        title: 'Sherlock Hormes',
+        author: 'Dr John Doe',
+        pages: '587',
+        type: 'Unread',
+      },
+      {
+        title: 'Tale of Two Cities',
+        author: 'Dr Smith Doe',
+        pages: '359',
+        type: 'Unread',
+      },
+    ];
+    const books = StoredBooks;
+    books.forEach(book => UI.addBookToList(book));
+  }
+
+  static addBookToList(book) {
+    const list = document.getElementById('tableBody');
+    const row = `<tr>
+    <td>${book.title}</td>
+    <td>${book.author}</td>
+    <td>${book.pages}</td>
+    <td>${book.type}</td>
+    <td><a href='#' class='btn btn-danger btn-sm delete'>X</a></td> 
+    </tr>`;
+    list.innerHTML += row;
+  }
```

Comment:
- This function is not adding a book to a list, it is just appending HTML elements to your HTML tag with the class of `tableBody`, when the instructions for this project tell us that we need to store all of our instances of Book inside of an array and render from there, instead of appending html elements. [As stated here](https://www.theodinproject.com/courses/javascript/lessons/library)
![image](https://user-images.githubusercontent.com/46115725/72382942-f5f06200-36df-11ea-9efd-2773b17968cf.png)

Reply:
Resolved.